tests/build

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+if [ ! -e ./image/manifest ] ; then
+	echo "Wrong directory"
+	exit 1
+fi
+
+# Fail if "rootfs" already exists. We delete it at the end.
+mkdir image/rootfs
+
+# Populate rootfs
+(cd inspect && go build)
+cp inspect/inspect image/rootfs/
+mkdir image/rootfs/dir{1,2}
+echo -n dir1 > image/rootfs/dir1/file
+echo -n dir2 > image/rootfs/dir2/file
+
+# Create base image
+../bin/actool build          --overwrite image rkt-inspect.aci
+
+# Create alternative images
+../bin/actool patch-manifest --overwrite --exec="/inspect --print-msg=Hello --exit-code=0"  rkt-inspect.aci rkt-inspect-exit0.aci
```

Comment:
This is OK for now but I think ideally we move towards generating these programmatically from go; the actool libraries should make this easy, if they don't already